packages/toolpad-components/src/List.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { createComponent, IteratorItem, IteratorRenderer } from '@mui/toolpad-core';
+import { Box, List as MuiList, ListItem, SxProps } from '@mui/material';
+import { SX_PROP_HELPER_TEXT } from './constants';
+
+export type ListProps = {
+  // eslint-disable-next-line react/no-unused-prop-types
+  items: IteratorItem[];
+  disablePadding?: boolean;
+  renderItems: IteratorRenderer;
+  sx?: SxProps;
+};
+
+function List({ renderItems, disablePadding = false, sx }: ListProps) {
+  return (
+    <MuiList disablePadding={disablePadding} sx={{ width: '100%', ...sx }}>
```

Comment:
It might be worth to consider if we shouldn't have a generic list construct where we could use any container as a list item and then pass children inside that container so that those children elements would be rendered by iterating through list array.

I.e. once we have `CardComponent` I would expect that I can use "list" generic component to render array of items as cards.

Since this isn't what was originally specified in the task feel free to ignore my comment. I just think it's worth to have this in mind for future

Reply:
On an unrelated note, I might add a generic `Container` component as a stripped-down version of our current `Paper` component - it follows the same kind of thinking.

Reply:
`Iterator` I think it something what I'm thinking about. As I mentioned I would expect that we should be able to take any group of component and render it in array, thinking a list of Cards and how could that be done.
The only minor issue that I see with `Iterator` is that it's very technical term and `List` might be a more natural name to what it is suppose to do